contentcuration/contentcuration/viewsets/feedback.py

```diff
@@ -0,0 +1,53 @@
+from rest_framework import permissions
+from rest_framework import serializers
+from rest_framework import viewsets
+from rest_framework.permissions import IsAuthenticated
+
+from contentcuration.models import BaseFeedback
+from contentcuration.models import BaseFeedbackEvent
+from contentcuration.models import BaseFeedbackInteractionEvent
+from contentcuration.models import FlagFeedbackEvent
+
+
+class IsAdminForListAndDestroy(permissions.BasePermission):
+    def has_permission(self, request, view):
+        # only allow list and destroy of flagged content to admins
+        if view.action in ['list', 'destroy']:
```

Comment:
Does `list` include `retrieve` (as in a single record)?

Reply:
I think we need to explicitly add the retrieve too! Thanks